pennylane/ops/op_math/sum.py

```diff
@@ -132,6 +132,8 @@ class Sum(CompositeOp):
         method (str): The graph coloring heuristic to use in solving minimum clique cover for
             grouping, which can be ``'lf'`` (Largest First) or ``'rlf'`` (Recursive Largest
             First). This keyword argument is ignored if ``grouping_type`` is ``None``.
+        grouping_indices (Optional[List[List[int]]]): Which terms can be computed at the same time. Sets
+           the corresponding property. Takes precedence over ``grouping_type`` and ``method``.
```

Comment:
Could move this, together with other small fixes, into the capture-templates PR or yet another capture-unrelated PR

Reply:
This was about when I started wanting to break out a separate templates PR.  So, yes, I'd be good to move this to another PR.

Though setting the grouping indices is really only an issue with `AbstractOperator`, which can't have properties set on it.

Reply:
Moved to #5524

Reply:
Should this be gone? :thinking: